.github/workflows/protocol-devchain-anvil.yml

```
@@ -34,13 +48,13 @@ jobs:
           fi
       - name: Foundry cache
         id: foundry-cache
-        uses: actions/cache@v4
+        uses: actions/cache@v3
         with:
           path: ./cache
           key: ${{ runner.os }}-foundry-cache-${{ env.FOUNDRY_CACHE_KEY }}
       - name: Foundry out
         id: foundry-out
-        uses: actions/cache@v4
+        uses: actions/cache@v3
```

Nit: We can keep this on v4 like it is on `master` since this small PR was merged last week:
- https://github.com/celo-org/celo-monorepo/pull/11087

```suggestion
        uses: actions/cache@v4
```